python/pyspark/sql/functions.pyi

```diff
@@ -57,7 +57,7 @@ def monotonically_increasing_id() -> Column: ...
 def nanvl(col1: ColumnOrName, col2: ColumnOrName) -> Column: ...
 def percentile_approx(
     col: ColumnOrName,
-    percentage: Union[Column, float, List[float]],
+    percentage: Union[Column, float, List[float], tuple[float]],
```

Comment:
Note for reviewer:
https://github.com/apache/spark/blob/ff1fc5ed4b685b4f5f83d5f600b24f089dd4522e/python/pyspark/sql/functions.py#L1218